taxi/forms.py

```diff
@@ -0,0 +1,31 @@
+from django import forms
+from django.contrib.auth import get_user_model
+
```

Comment:
You should place a blank line after the `from django.contrib.auth.forms import UserCreationForm` import